lib/ret_web/router.ex

```diff
@@ -69,6 +69,12 @@ defmodule RetWeb.Router do
     forward("/", RetWeb.Plugs.PostgrestProxy)
   end
 
+  scope "/api/ita/configs" do
+    pipe_through([:secure_headers, :admin_required])
+    post("/app-config", RetWeb.Api.V1.AppConfigController, :create)
```

Comment:
sorry I poorly phrased my comment wrt ita, what I was getting at is the json protocol for the app configs should be the same as ita's. for example, if we had a hubs config called foo.bar it should be POSTed in the form:
```
{
hubs: {
   foo: {
      bar: "value"
   }
}
```

i didn't mean we should embed the app config REST under the `/api/ita` path -- i think `/api/ita` should strictly be a proxy to the underlying `ita` service. (whose `POST` API for configs is the same as our app level one as above)